pkg/spoke/controllers/manifestcontroller/manifestwork_controller.go

```diff
@@ -187,18 +188,41 @@ func (m *ManifestWorkController) sync(ctx context.Context, controllerContext fac
 		manifestCondition.Conditions = append(manifestCondition.Conditions, buildAppliedStatusCondition(result))
 
 		newManifestConditions = append(newManifestConditions, manifestCondition)
+
+		if apierrors.IsForbidden(result.Error) {
+			// If it is a forbidden error, after the condition is constructed, we set the error to nil
```

Comment:
I think we should let validator to return the requeue time in the error. We can construct a special error struct in validator. Here we get the requeue time for each manifest in the work, and find the min one. This will be usefule when we implement validate cache later. Because the validator should use cache ttl to calculate the requeue time.